charts/prometheus-cloudwatch-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.8.0"
 description: A Helm chart for prometheus cloudwatch-exporter
 name: prometheus-cloudwatch-exporter
-version: 0.12.1
+version: 0.12.2
```

Comment:
Can you change it to 0.13.0. As it's a new feature it's worth incrementing the minor version.

Reply:
No worries! Bumped and also fixed the version in the `CHANGELOG.md` as realised i'd mistyped that.